src/Microsoft.AspNet.OData.Shared/ClrEnumMemberAnnotation.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.OData.Common;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Represents a mapping from an <see cref="IEdmEnumMember"/> to a CLR Enum member.
+    /// </summary>
+    public class ClrEnumMemberAnnotation
```

Comment:
>ClrEnumMemberAnnotation [](start = 17, length = 23)

Looks like there are two maps used, for regular lookup and reversed lookup. Not sure what is the perf gain for this? Look like the original _maps is only used in one place for serialization.
Also please update the summary to reflect the direction(s) of the lookup.